src/applications/post-911-gib-status/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "Post 9/11 GI Bill Status",
+  "entryFile": "./post-911-gib-status-entry.jsx",
+  "entryName": "post-911-gib-status",
+  "rootUrl": "/education/gi-bill/post-9-11/ch-33-benefit",
+  "production": true
+}
```

Comment:
This wasn't necessary, but I did it anyway as I was exploring ideas for rendering React w/a content page as its root url.